decentraland/unity-explorer
# fix: avoid nametag with inconsistend messages

## What does this PR change?

Fix #2169 
Fix #1774 
This PR fixes the issue that caused other user messages to be displayed in our own nametag chat bubble
...

## How to test the changes?



1. Launch the explorer
2. Chat and check that nametags and chat bubble are on the correct user

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
